### PR TITLE
Properly determine kernel affinity in controller

### PR DESCRIPTION
**Pull request by pacevedom**

### CODE REVIEW

File: controllers/resources.go [hunk 1]
```diff
@@ -19,6 +19,10 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 )
 
+var (
+	affineRegex = regexp.MustCompile(`\n\s+specialresource\.openshift\.io\/kernel\-affine`)
```
Comment (qbarrand) [inline controllers/resources.go:23]: I'm missing if the `\n` is there on purpose. If not, how about:
```suggestion
	affineRegex = regexp.MustCompile(`^\s+specialresource\.openshift\.io/kernel-affine:.+$`)
```
I think some escapes might be redundant.

Comment (pacevedom) [reply controllers/resources.go]: It was on purpose, but you got in my head the multiline flag that I forgot in the first place. Thanks!

**Pull request merged**